src/components/notesCard/NotesCard.jsx

```diff
@@ -26,13 +30,18 @@ const NotesCard = (noteModal) => {
 
                 </div>
                 <div className="card__notes__actions__container flex jc-space-around">
-                    <FontAwesomeIcon icon={faArchive} />
-                    <FontAwesomeIcon icon={faRecycle} />
-                    <FontAwesomeIcon icon={faTrash} onClick={() => deleteFromNotes(dispatch_note, noteModal._id, token)} />
-                    <FontAwesomeIcon onClick={() => setIsModalOpenUpdate(true)} icon={faPen} />
+
+                    {archive || trash || <FontAwesomeIcon icon={faArchive} onClick={() => archiveHandler()} />}
```

Comment:
didn't get why are you using trash flag in showing archive Icon

Reply:
Trash flag is actually there so that archive icon does'nt get rendered i am using OR short-circuiting if trash flag or archive flag is there that icon won't get rendered, iam using the same card in notes, archives and, trash so to make it resuable.